templates/spark/v1/spark.jinja

```diff
@@ -0,0 +1,112 @@
+{% set WORKERS = properties['workers'] if properties and properties['workers'] else 2 %}
```

Comment:
I presume you mean `= PROPERTIES['foo']` there? But yeah, that looks like a neat pattern.